lib/import/import_samples.rb

```diff
@@ -177,6 +177,15 @@ def molecule_not_exist(molecule)
     molecule.nil?
   end
 
+  # format row[field] for melting and boiling point
+  def process_row(row_field)
```

Comment:
i would rename method by a more meaningful name, maybe format_to_interval_syntax or something like that